packages/js/product-editor/src/components/require-password/require-password.tsx

```diff
@@ -67,7 +67,11 @@ export function RequirePassword( {
 					<InputControl
 						id={ postPasswordId }
 						value={ postPassword }
-						onChange={ onInputChange }
+						onChange={ ( value ) => {
+							if ( value ) {
+								onInputChange( value );
+							}
+						} }
```

Comment:
```suggestion
						onChange={ ( value ) => {
							onInputChange( value ?? '' );
						} }
```
This breaks the `password` field, and actually cause the last character to always be present.
The above would be better.

Reply:
Fixed with e1f833ffa51d6d0e7100adcce0347930fee5be5f